cmd/image.go

```diff
@@ -115,18 +116,22 @@ func compressImage(collage image.Image, quality int) (image.Image, error) {
 	return collageCompressed, nil
 }
 
-func placeText(img *image.RGBA, album Album, displayArtist bool, displayAlbum bool) {
+func placeText(img *image.RGBA, album Album, displayArtist bool, displayAlbum bool, playCount bool) {
```

Comment:
Should probably rename this to like `displayPlaycount` or something in the future to be more consistent